src/main/java/uk/ac/ebi/eva/utils/MongoDBHelper.java

```diff
@@ -86,8 +86,4 @@ public static String buildStorageId(String chromosome, int start, String referen
         return builder.toString();
     }
 
-    public static String buildStorageFileId(String studyId, String fileId) {
```

Comment:
I would suggest leaving the code split in `MongoDBHelper` and `URLHelper`, as they previously were. The fact that their implementations currently match doesn't mean they are the same, but they apply to very different contexts. It is not intuitive to look in `URLHelper` if to get a Mongo key.